ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/Coordinate.java

```diff
@@ -3,7 +3,7 @@
 public class Coordinate extends Model<Coordinate> {
 	private double latitude;
 	private double longitude;
-	
+	public static final Coordinate NORTH_KOREA_CONCENTRATION_CAMP_COORDINATES = new Coordinate(39.03808, 125.7296);
```

Comment:
This is kinda funny, but I'm concerned of possibly running afoul of university Code of Conduct rules, North Korea and concentration camps both being touchy subjects. I would recommend to just name this `DEFAULT_COORDINATES`, which is both a shorter and more obvious name anyway.